ee/codegen/src/__test__/__snapshots__/workflow.test.ts.snap

```diff
@@ -9,17 +9,6 @@ class TestWorkflow(BaseWorkflow):
 "
 `;
 
-exports[`Workflow > write > should generate correct code when there are input variables with escape characters 1`] = `
```

Comment:
looks unrelated and an issue on main - will merge through separately to keep this PR focused